stacks/container-apps/play.prx.org.yml

```diff
@@ -201,6 +203,159 @@ Resources:
           Value: !Ref PlatformALBFullName
         - Name: TargetGroup
           Value: !GetAtt PlayALBTargetGroup.TargetGroupFullName
+
+  ProxyPathListenerRule:
+    Type: AWS::ElasticLoadBalancingV2::ListenerRule
+    Condition: CreateStagingResources
+    Properties:
+      Actions:
+        - TargetGroupArn: !Ref ProxyTargetGroup
+          Type: forward
+      Conditions:
+        - Field: host-header
+          Values:
+            - play.*
+        - Field: path-pattern
+          Values:
+            - /proxy*
+      ListenerArn: !Ref PlatformALBHTTPSListenerArn
+      Priority: !Join ["", [!Ref PlatformALBListenerPriorityPrefix, "20"]]
+
+  ProxyTargetGroup:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    DependsOn:
+      - ProxyFunctionAlbPermission
+    Properties:
+      HealthCheckEnabled: false
+      Tags:
+        - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+        - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+        - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+        - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+        - { Key: prx:dev:family, Value: Dovetail }
+        - { Key: prx:dev:application, Value: Play }
+      Targets:
+        - Id: !GetAtt ProxyFunction.Arn
+      TargetType: lambda
+
+  ProxyFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Description: !Sub ${EnvironmentType} Play feed proxy
+      Environment:
+        Variables:
+          MAX_HTTP_REDIRECTS: 4
+      Handler: index.handler
+      InlineCode: |
+        const http = require("http");
+        const https = require("https");
+
+        const PROXY_UA = "play.prx.org feed proxy";
+        const ACCEPTED_TYPES = [
+          "application/rss+xml",
+          "application/rdf+xml;q=0.8",
+          "application/atom+xml;q=0.6",
+          "application/xml;q=0.4",
+          "text/xml;q=0.4",
+        ];
+
+        async function request(url, redirectCount, redirectUrl) {
+          return new Promise((resolve, reject) => {
+            const q = new URL(redirectUrl || url);
+
+            // Setup request options
+            const options = {
+              host: q.host,
+              port: q.port,
+              path: `${q.pathname || ""}${q.search || ""}`,
+              method: "GET",
+              headers: {
+                Accept: ACCEPTED_TYPES.join(","),
+                "User-Agent": PROXY_UA,
+              },
+            };
+
+            const h = q.protocol === "https:" ? https : http;
+            const req = h.request(options, (res) => {
```

Comment:
I don't think it matters, but the Play proxy currently uses the `request` library.  Which would have followed any redirects.  But in practice ... I can't imagine anyone is (or should be) using the player with a redirect url.

Reply:
Oh wait, nevermind!  You do follow redirects below.